schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id              INT,
+    name            CHAR(20),
+    date_of_birth   DATE,
+    escape_attempts INT, 
+    neutered        BOOLEAN,
+    weight_kg       DECIMAL
 );
```

Comment:
## [OPTIONAL]

- [ ] Nice work so far 👏 , however, you can improve this by making the id column not null and autoincremented. Hint 👇 

```.sql
    id int NOT NULL IDENTITY(1,1)
```